### DISCUSSION TITLE
TVSeries not showing option to restart or resume episode

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/DP2SceneGraphWizard#43] [Q&A] [answered]
I thought I had posted this, but don't see it.

When I side loaded under beta 6, the option to resume or restart the one (only one posted so far) TV Episode was showing. Channel is now published (using beta 7), but the option ONLY on the TV episode is no longer showing (works for shortFormVideos).

Would this happen if I had a mismatch in the Roku Dashboard Deep Linking section for the ID on the one TV episode I had entered and the actual ID in the feed? If so, is there some way to SEE the number I ended into the Dashboard Deep Linking section so that I can match the ID in my feed? (Dashboard doesn't allow access after the channel has published).

I'm not marking this as an issue since I think it might be on my end!

**Comment by ferdiworks** [comment]
**YOU ASKED is there some way to SEE the number I ended into the Dashboard Deep Linking section so that I can match the ID in my feed?**
@adamesq Yes you can sign into the Roku DEVELOPER DASHBOARD, go to MY CHANNELS, and select your channel. Then from the dropdown select DEEP LINKING and scroll down to see the video and info you deep linked!

Watch this video
https://www.loom.com/share/7c05df66ea074928a38d790b63bd528e?sid=7172c742-42e9-435c-8761-237d68c193ab

**Reply by adamesq** [reply]
Thank you so much, Joe. The problem is that the channel _published_. The Deep Linking section is blanked-out once the channel publishes. The message now in the Deep Linking Section now reads:
"This page is not available until you upload a new package."

I could be chasing a red herring here anyway while trying to figure out what is causing the TVSeries episode to NOT have the resume/restart option while the shortformvideos DO.

**Reply by rrirower** [reply]
The only way you'll see resume/restart is if you watch a video and stop before the end.

Send me your URL and the name of the video you expect to see with those options.

**Reply by rrirower** [reply, marked as answer]
Restart/Resume is currently not available on a Series with Episodes.  That will have to be added as an enhancement.

**Reply by rrirower** [reply]
Can you detail what you'd like to see?  Given that a Series uses 'smart' bookmarks to remember the last Episode and playback position, what would you want to see on the screen? I'm trying to avoid clogging up the screen for a Series with Episodes or Seasons.

**Reply by adamesq** [reply]
I see what you're saying. For a an episode of a season, that would seem to require an extra click...where after selecting an episode, an  option to start or continue would be presented. 🤔 I'd prefer it, but quite possible others would not. A progress bar would at _least_ indicate the episode has been started (indicating where the viewer is in the series). 

I supposed a work-around would be just listing each episode as a special in the same category, with the category being the name of the series.

**Comment by rrirower** [comment]
New feature added in 1.0.02 to restart Episodes.
